collectors/utils.py

```diff
@@ -98,3 +99,27 @@ def handle_urls(references: list, source_ref: str) -> list:
             pass
 
     return urls
+
+
+CVSS_SCORE_TO_IMPACT = {
```

Comment:
Is this modeled after https://www.first.org/cvss/specification-document#Qualitative-Severity-Rating-Scale ?

Reply:
Partially yes. It matches both Impact in OSIDB and CVSS severity defined by FIRST. However, FIRST does not define severity for CVSSv2, so I hope this simplification is good.